src/tests/app.rs

```diff
@@ -5,11 +5,12 @@ use crate::test_helpers::echo::EXECUTE_REPLY_BASE_ID;
 use crate::test_helpers::{caller, echo, error, hackatom, payout, reflect, CustomMsg};
 use crate::transactions::{transactional, StorageTransaction};
 use crate::wasm::ContractData;
+use crate::AppBuilder;
```

Comment:
Why move this up here if it's only used in tests?

Reply:
This file contains only tests. So all imports automatically are at the top level. `cargo fmt` sorts them out.